# Tache.msg
int32 task_id
int32 mission_id
string task_type
geometry_msgs/Pose goal
